LDM-562.tex

```diff
@@ -0,0 +1,357 @@
+
+\subsection{Wavelength Dependence}
+
+\label{DMS-L2JC-REQ-0005}
+\textbf{ID:} DMS-L2JC-REQ-0005
+
+\textbf{Specification:}
+All JointCal photometric model components shall have a wavelength-dependent form defined by a linear combination of wavelength-independent model components of the same type.
+
+\textbf{Discussion:}
+Sub-band SEDs for objects will be defined by a small number of orthogonal linear parameters (either simple bins or a series expansion), and will be inferred from object colors via a call to an external (to JointCal) function.  The set of sub-band SED parameters can then be used (after scaling by the unknown total fluxes of objects) to constrain wavelength-dependent model components in the same way measured object fluxes constrain wavelength-independent model components.
+
+\subsection{Group-Based Component Variation}
+
+\label{DMS-L2JC-REQ-0006}
+\textbf{ID:} DMS-L2JC-REQ-0006
+
+\textbf{Specification:}
+It shall be possible to tie any JointCal model component's variation to a group of one or more sensors, a group of one or more exposures, or a group of exposure-sensor pairs.
+
+\textbf{Discussion:}
+Examples for groups of sensors may include rafts or sensors produced by a particular vendor.  Examples of groups of exposures may include those observed with a particular filter or those observed in a particular time range.  Many groups will simply contain a single sensor or exposure.  Full-focal plane model components will never be tied to individual sensors, of course.  A design that could be extended to support different parameters to different amplifiers would be highly desirable in case this is needed in the future.
+
+\subsection{Photometric Jacobian Component}
+
+\label{DMS-L2JC-REQ-0007}
+\textbf{ID:} DMS-L2JC-REQ-0007
+
+\textbf{Specification:}
+JointCal shall include in the photometric model a correction for the variation in pixel area determined from the astrometric model.
+
+\textbf{Discussion:}
+Like other components in the photometric fit, this correction should be optional; in some contexts this correction may be included in the catalogs used as input instead.
+
+\subsection{Photometric Spatially-Varying Components}
+
+\label{DMS-L2JC-REQ-0008}
+\textbf{ID:} DMS-L2JC-REQ-0008
+
+\textbf{Specification:}
+JointCal shall include photometric model components that vary smoothly over either the full focal plane or individual devices.
```

Comment:
@RobertLuptonTheGood says:
> We need to be careful about "smoothly".  It isn't clear that all ghosts and filter effects will be smooth; if it significantly simplifies the code smooth is probably OK (but I'd hope to allow wavelet expansions if appropriate).

This certainly doesn't disallow wavelets - it just doesn't require them (and I have zero concern that we're designing away the possibility of including them if we need them).  Is that sufficient?

How do you envision us actually using wavelets here?  I assume we'd significantly reduce the degrees of freedom in any wavelet-based model for ghosts or filter variation by some external-to-jointcal procedure, and then just fit a few linear parameters?